src/algebra/group/with_one.lean

```diff
@@ -292,15 +294,18 @@ instance [monoid α] : monoid_with_zero (with_zero α) :=
 instance [comm_monoid α] : comm_monoid_with_zero (with_zero α) :=
 { ..with_zero.monoid_with_zero, ..with_zero.comm_semigroup }
 
-/-- Given an inverse operation on `α` there is an inverse operation
-  on `with_zero α` sending `0` to `0`-/
-instance [has_inv α] : has_inv (with_zero α) := ⟨λ a, option.map has_inv.inv a⟩
+/-- When `α` has multiplicative inverses, `with_zero α` also has multiplicative inverses, by
+defining `0⁻¹ = 0`. -/
+@[to_additive] instance [has_inv α] : has_inv (with_zero α) := ⟨λ a, option.map has_inv.inv a⟩
```

Comment:
By adding a `to_additive` docstring, we can make clear what the effect of `@[to_additive]`ing is supposed to be.
```suggestion
defining `0⁻¹ = 0`. -/
@[to_additive "When `α` has additive inverses, `with_zero α` also has additive inverses,
by defining `-0 = 0`."]
instance [has_inv α] : has_inv (with_zero α) := ⟨λ a, option.map has_inv.inv a⟩
```

Reply:
even though this docstring has already been set erlier by a previous `to_additive`?